docs/reference.md

```diff
@@ -0,0 +1,73 @@
+# Reference
+
+---
+
+This part of the project documentation focuses on
```

Comment:
`radCAD`. What is an "information-oriented" approach? I think if we're going to use these definitions (not sure that we need to explicitly say why we structured it in a certain way), then we need to define the terms.